common/src/main/java/com/wynntils/screens/lists/OverlayList.java

```diff
@@ -27,6 +28,10 @@ public class OverlayList extends ContainerObjectSelectionList<OverlayEntry> {
             new TextComponent("Left click on the overlay to edit it."),
             new TextComponent("Right click on the overlay to disable/enable it."));
 
+    private static final List<Component> DISABLED_PARENT_TOOLTIP_LINES = List.of(
```

Comment:
Shouldn't these be translatable?

Reply:
These were never translatable. I don't plan on fixing this rn because of the resource manager init thing we have not sorted out yet.

Reply:
@HighCrit We can fix this after https://github.com/Wynntils/Artemis/pull/294